app-web/nheko/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=nheko
+PKGSEC=web
+PKGDEP="qt-6 mtxclient lmdb lmdb++ cmark libolm spdlog fmt gstreamer libnice opus srtp libxcb xcb-util-wm qtkeychain-6 kdsingleapplication openssl glibc asciidoc"
```

Comment:
Why was `asciidoc` a runtime dependency (it is also recommended to use `asciidoctor`)?